README.md

```diff
@@ -70,7 +70,7 @@ Example:
 ```lua
 -- general
 lvim.format_on_save = true
-lvim.colorscheme = "onedarker"
+lvim.colorscheme = "tokyonight"
```

Comment:
```suggestion
lvim.colorscheme = "tokyonight-night"
```

Reply:
but we should at least name it correctly no?

like you can do 
```lua
lvim.colorscheme = "tokyonight"
lvim.builtin.theme.options.style = "night"
```